test/trip_dispatch_test.rb

```diff
@@ -1,12 +1,12 @@
-require_relative 'test_helper'
+require_relative "test_helper"
```

Comment:
Bit of a nitpick, but the naming convention for these files should match the class that you're testing, so instead of `trip_dispatch_test.rb`, maybe `trip_dispatcher_test.rb`